src/pages/dashboard/hooks/useDashboardData.ts

```diff
@@ -231,33 +231,29 @@ export const useDashboardData = (filters: any) => {
         value: projectFullDto.treesPlantedCount ?? 0,
         totalValue: projectFullDto.treesGrownGoal ?? 0
       });
-    } else if (totalSectionHeaderData?.data) {
+    } else if (totalSectionHeader) {
       setDashboardHeader(prev => [
         {
           ...prev[0],
-          value: totalSectionHeaderData?.data?.totalTreesRestored
-            ? totalSectionHeaderData?.data?.totalTreesRestored.toLocaleString()
-            : "-"
+          value: totalSectionHeader?.totalTreesRestored ? totalSectionHeader?.totalTreesRestored.toLocaleString() : "-"
```

Comment:
In this block, you should be able to drop the optional chaining, and also remember to use explicit null checks:
```
value: totalSectionHeader.totalTreesRestored != null ? 
```

Reply:
In a case like this, it can even be streamlined a little more:
```
value: totalSectionHeader.totalTreesRestored?.toLocaleString() ?? "-"
```